README.rst

```diff
@@ -1711,6 +1712,20 @@ Choose from the list of available rules:
 
   *Risky rule: risky when the ``settype`` function is overridden or when used as the 2nd or 3rd expression in a ``for`` loop .*
 
+* **short_echo_tag** [@PhpCsFixer]
+
+  Replace short-echo ``<?=`` with long format ``<?php echo``/``<?php print``
+  syntax, or vice-versa.
+
+  Configuration options:
+
+  - ``format`` (``'long'``, ``'short'``): how the fixer should process short/long echo
```

Comment:
The description could be more clear, like from `no_mixed_echo_print` - `the desired language construct`.

Reply:
:+1: 94dc2c3dd377c5c08cd7c04c91a5ec1864bb20c5